src/webgl/framebuffer.js

```diff
@@ -295,7 +297,7 @@ export default class Framebuffer extends Resource {
       // Allocate pixel array if not already available, using supplied type
       type = type || gl.UNSIGNED_BYTE;
       const ArrayType = getTypedArrayFromGLType(type, {clamped: false});
-      const components = glFormatToComponents(format);
+      const components = _glFormatToComponents(format);
```

Comment:
This method is for converting `format` values specific to `readPixels` API, and not exported , so make it a private helper method.